exporters/InventorRobotExporter/InventorRobotExporter/Exporter/SkeletonBuilder.cs

```diff
@@ -39,6 +37,10 @@ public static RigidNode_Base ExportSkeleton(IProgress<ProgressUpdate> progress)
                 WinFormsUtils.ShowErrorDialog("Please ground a part in your assembly to export your robot.", "No Ground");
                 return null;
             }
+
+            //Building Model...
```

Comment:
Why was this progress report moved?

Reply:
I think I was experimenting with the progress report to see whether I could get rid of the notification, and I left it there. There aren't any noticeable effects from what I can see, but I'll move it back.